sdk/communication/AzureCommunicationChat/Source/Generated/Chat.swift

```diff
@@ -29,16 +29,16 @@ public final class Chat {
     ///    - options: A list of options for the operation
     ///    - completionHandler: A completion handler that receives a status code on
     ///     success.
-    public func create(
-        chatThread: CreateChatThreadRequest,
+    internal func create(
+        chatThread: CreateChatThreadRequestInternal,
         withOptions options: CreateChatThreadOptions? = nil,
-        completionHandler: @escaping HTTPResultHandler<CreateChatThreadResult>
+        completionHandler: @escaping HTTPResultHandler<CreateChatThreadResultInternal>
     ) {
         let dispatchQueue = options?.dispatchQueue ?? client.commonOptions.dispatchQueue ?? DispatchQueue.main
 
         // Create request parameters
         let params = RequestParameters(
-            (.header, "repeatability-request-id", options?.repeatabilityRequestId, .encode), (
+            (.header, "repeatability-Request-Id", options?.repeatabilityRequestId, .encode), (
```

Comment:
Investigate if this is a Swagger issue or autorest.

Reply:
It's generated by autorest

Reply:
This is a Swagger issue. 
https://github.com/Azure/azure-rest-api-specs/blob/d95c18e2d5fc678a1453c454d746fdce22d30122/specification/communication/data-plane/Microsoft.CommunicationServicesChat/stable/2021-03-07/communicationserviceschat.json#L853
Autorest is generating what the Swagger specifies, so it needs to be fixed there.

Reply:
Updated, I reran the autorest, verified its in lowercase.